dashboard-input/undo-redo/demo.html

```diff
@@ -1,7 +1,6 @@
 <script type='text/javascript' src="https://mobiledemo.jaspersoft.com/jasperserver-pro/client/visualize.js"></script>
 
-<!-- JQuery - compiled and minified -->
-<script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
+
```

Comment:
so we are not using jquery anymore in examples right?
just want to confirm

Reply:
We are providing jquery with fiddle itself. for example below URL will automatically include jquery.
https://jsfiddle.net/gh/get/jQuery/3.4.1/anup1998/JS-visualize/tree/master/report-export/json-export-d3/